src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -107,7 +107,8 @@ protected ActionResult execute() {
             log.severe("Instructor welcome email failed to send: " + TeammatesException.toStringWithStackTrace(e));
         }
         data.statusForAjax = "Instructor " + SanitizationHelper.sanitizeForHtml(data.instructorName)
-                             + " has been successfully created with join link:<br>" + joinLink;
+                             + " has been successfully created and will receive an email shortly with "
+                             + "the below join link.<br><a href=\"" + joinLink + "\">join link</a>";
```

Comment:
Actually I prefer the original phrasing, which is more concise. Why not just link the existing words? `... has been successfully created with <a href="...">join link</a>.`